container.go

```diff
@@ -49,6 +49,7 @@ type Container interface {
 	NetworkAliases(context.Context) (map[string][]string, error) // get container network aliases for a network
 	Exec(ctx context.Context, cmd []string) (int, error)
 	ContainerIP(context.Context) (string, error) // get container ip
+	CopyFileToContainer(ctx context.Context, containerPath string, fileName string, fileContent []byte, fileMode int64) error
```

Comment:
What do you think about not exposing the `fileContent` in method header, and obtain it internally? That would simplify clients code.

Reply:
that sounds great, do you think a single parameter like filePath, meaning the filePath on the client side, and from it we can extract the filename to save in the container?

or maybe have a containerFilePath (instead of containerPath) and from it extract the name to save in the container?

What do you think?

Reply:
I like the first one more: containerFilePath and hostFilePath. I'd get the file name from the host file, using it in the container too. Wdyt?

Besides that, what are the Java folks doing? (I'm currently on the phone)

Reply:
In java the method receives a Transferable (I imagine that this is the content in bytes) and the filepath that we want in the container (path + filename). https://github.com/testcontainers/testcontainers-java/blob/6bdbc8ffe257fc879791eb7fddcf3b5ecccdd52d/core/src/main/java/org/testcontainers/containers/ContainerState.java#L262

Regarding the suggested solution, I think it would be better to use the filename from the container one, this way users will be able to copy the file to the container and at the same time rename it, its more flexible. What do you think?

Reply:
I updated the interface, let me know if it sounds good to you